pkg/shell/indexes.js

```diff
@@ -855,6 +545,7 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
     });
 }
 
+// TODO - this cannot work
 function SimpleIndex(index_options) {
```

Comment:
I faintly remember discussing the removal of the simple index. If it doesn't work any more, let's kill it properly.